std_msgs/Float64 gripper_state # 0 o 239
geometry_msgs/PoseStamped tcp_pose
######## geometry_msgs/TwistStamped tcp_velocity
geometry_msgs/WrenchStamped tcp_force_torque      # Array di immagini delle telecamere